src/main/kotlin/uk/gov/justice/digital/hmpps/breachnoticeapi/controller/BreachNoticeController.kt

```diff
@@ -104,4 +104,33 @@ class BreachNoticeController(private val breachNoticeService: BreachNoticeServic
     ],
   )
   fun updateBreachNotice(@PathVariable id: UUID, @RequestBody breachNotice: BreachNotice) = breachNoticeService.updateBreachNotice(id, breachNotice)
+
+  @GetMapping("/{uuid}/pdf")
+  @Tag(name = "Breach Notice")
+  @Operation(
+    summary = "Retrieve a breach notice pdf by uuid - breach notice id",
+    description = "Calls through the breach notice service to retrieve a generate ",
+    security = [SecurityRequirement(name = "breach-notice-api-ui-role")],
+    responses = [
+      ApiResponse(responseCode = "200", description = "breach notice pdf returned"),
+      ApiResponse(
+        responseCode = "401",
+        description = "Unauthorized to access this endpoint",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
+      ),
+      ApiResponse(
+        responseCode = "403",
+        description = "Forbidden to access this endpoint",
+        content = [Content(mediaType = "application/json", schema = Schema(implementation = ErrorResponse::class))],
+      ),
+    ],
+  )
+  fun getBreachNoticeAsPdf(@PathVariable uuid: UUID): ResponseEntity<ByteArray> {
+    var breachNotice: BreachNoticeDetails? = breachNoticeService.getBreachNoticeById(uuid)
```

Comment:
Should we return a 404 here if the breachNoticeDetails is null?
```suggestion
    var breachNotice = breachNoticeService.getBreachNoticeById(uuid) ?: throw NotFoundException("Breach notice", "id", uuid)
```